### ISSUE TITLE
Wrow, Wcol

### ISSUE THREAD

**Issue by snsatoshi** [fmonti/mgcnn#1]
How did you generate Wrow and Wcol that are contained in mat file ?

**Comment by fmonti** [comment]
As stated in the paper:

For the MovieLens dataset we constructed the user and item (movie) graphs as unweighted 10-nearest
neighbor graphs in the space of user and movie features, respectively. For Flixster, the user and item graphs were constructed from the scores of the original matrix (so still it's a nearest neighbors graph). For the Douban dataset, we used only the user graph (i.e. the provided social network of the user). For the YahooMusic dataset, we used only the item graph, constructed with unweighted 10-nearest neighbors in the space of item features (artists, albums, and genres).

**Comment by jiaxinwoo** [comment]
Hi,
For MovieLens dataset，what exactly are the user and movie features?

**Comment by fmonti** [comment]
You can find all the needed information in the dataset README:

http://files.grouplens.org/datasets/movielens/ml-100k-README.txt

**Comment by jianing-sun** [comment]
@fmonti if you use 10-nearest neighbor, should you have 10 links for each user? But in your `Wrow` matrix, you have different neighbor numbers for each user, e.g. 14, 27, 13... 
Did I miss anything? thanks

**Comment by fmonti** [comment]
We used 10-NNs only when the original graph was not provided.

**Comment by jianing-sun** [comment]
> We used 10-NNs only when the original graph was not provided.

Yes I understand. But how could you release more details regarding building graphs on user and item respectively, e.g. what's the user/item input feature exactly, and how did you compute the similarity, cosine? 

Thanks!
